### ISSUE TITLE
TextFormat prints fields in alphabetical order

### ISSUE THREAD

**Issue by fkm3** [google/proto-lens#40]
Given this protobuf

``` protobuf
message Date {
  int32 year = 1;
  int32 month = 2;
  int32 day = 3;
}
```

you get

``` plain
day: 1 month: 9 year: 2016
```

I think it would be better to order by tag number.

**Comment by judah** [comment]
Note: it looks like this wasn't actually fixed by PR #48; probably that reference to this issue was a typo.

**Comment by fkm3** [comment]
Thanks!
